pkg/cmd/cli/cmd/set/triggers.go

```diff
@@ -512,6 +522,44 @@ func defaultNamespace(namespace, defaultNamespace string) string {
 	return namespace
 }
 
+// NewAnnotationTriggers creates a trigger definition from an object that can be triggered by the image
+// annotation.
+func NewAnnotationTriggers(obj runtime.Object) (*TriggerDefinition, error) {
+	m, err := meta.Accessor(obj)
+	if err != nil {
+		return nil, err
+	}
+
+	t := &TriggerDefinition{ConfigChange: true}
+	switch typed := obj.(type) {
+	case *kextensions.Deployment:
+		t.ConfigChange = !typed.Spec.Paused
```

Comment:
ConfigChange does not equal Paused. You can have a DC that only uses an ICT and it will deploy on an image update whereas Paused gates every update. This shouldn't be a problem if we were able to start one-off rollouts from paused Deployments.